shotover/src/hot_reload/client.rs

```diff
@@ -43,7 +45,7 @@ impl UnixSocketClient {
         }
     }
 
-    async fn send_request_inner(&self, request: Request) -> Result<Response> {
+    async fn send_request_inner(&self, request: Request) -> Result<HashMap<u16, TcpListener>> {
```

Comment:
Its worth explicitly calling out that having this function return a specific response instead of a generic `Response` enum is not ideal. It kind of confuses the "request in, response out" abstraction used here.
We will want to have a new Response type that can contain any kind of response, including this HashMap response.

That said, we also only have a single kind of response so far, so hard to predict what our needs will be for a response type.
So I think its best to just wait till we implement a new request/response type in the next chunk of work.

No changes needed.
You can just resolve this comment when its time to merge.